contracts/minter-suite/Minters/MinterSEAV0.sol

```diff
@@ -0,0 +1,987 @@
+     * @dev Note that a `_maxInvocations` of 0 can only be set if the current `invocations`
+     * value is also 0 and this would also set `maxHasBeenInvoked` to true, correctly short-circuiting
+     * this minter's purchase function, avoiding extra gas costs from the core contract's maxInvocations check.
+     * @param _projectId Project ID to set the maximum invocations for.
+     * @param _maxInvocations Maximum invocations to set for the project.
+     */
+    function manuallyLimitProjectMaxInvocations(
+        uint256 _projectId,
+        uint256 _maxInvocations
+    ) external {
+        _onlyArtist(_projectId);
+        // CHECKS
+        // ensure that the manually set maxInvocations is not greater than what is set on the core contract
+        uint256 maxInvocations;
+        uint256 invocations;
+        (invocations, maxInvocations, , , , ) = genArtCoreContract_Base
+            .projectStateData(_projectId);
+        require(
+            _maxInvocations <= maxInvocations,
+            "Cannot increase project max invocations above core contract set project max invocations"
+        );
+        require(
+            _maxInvocations >= invocations,
+            "Cannot set project max invocations to less than current invocations"
+        );
+        // EFFECTS
+        // update storage with results
+        projectConfig[_projectId].maxInvocations = uint24(_maxInvocations);
+        // We need to ensure maxHasBeenInvoked is correctly set after manually setting the
+        // local maxInvocations value.
+        projectConfig[_projectId].maxHasBeenInvoked =
+            invocations == _maxInvocations;
+
+        emit ProjectMaxInvocationsLimitUpdated(_projectId, _maxInvocations);
+    }
+
+    /**
+     * @notice Sets the minimum and maximum values that are settable for
+     * `durationSeconds` for all project configurations.
+     * Note that the auction duration is the initial duration of the auction,
+     * and does not include any extensions that may occur due to new bids being
+     * placed near the end of an auction.
+     * @param _minAuctionDurationSeconds Minimum auction duration in seconds.
+     * @param _maxAuctionDurationSeconds Maximum auction duration in seconds.
+     */
+    function updateAllowableAuctionDurationSeconds(
+        uint256 _minAuctionDurationSeconds,
+        uint256 _maxAuctionDurationSeconds
+    ) external {
+        _onlyCoreAdminACL(this.updateAllowableAuctionDurationSeconds.selector);
+        // CHECKS
+        require(
+            _maxAuctionDurationSeconds > _minAuctionDurationSeconds,
+            "Only max gt min"
+        );
+        require(_minAuctionDurationSeconds > 0, "Only min gt 0");
+        // EFFECTS
+        minAuctionDurationSeconds = _minAuctionDurationSeconds.toUint32();
+        maxAuctionDurationSeconds = _maxAuctionDurationSeconds.toUint32();
+        emit AuctionDurationSecondsRangeUpdated(
+            _minAuctionDurationSeconds.toUint32(),
```

Comment:
nit: potentially worth just passing in `minAuctionDurationSeconds` & `maxAuctionDurationSeconds` to avoid extra `toUint32` function calls?

Reply:
i guess i'm also curious why you can't just intake uint32 (instead of uint256) here for the external function arguments, given that you do that below with `updateMinterTimeBufferSeconds`

Reply:
absolutely can do this 🙏 good call, updated in 8389aa1